examples/deploy-virtual-appliance-from-marketplace/main.tf

```diff
@@ -0,0 +1,77 @@
+provider "azurerm" {
```

Comment:
Could we use this format as found in some other examples?

```hcl
# Configure the Microsoft Azure Provider
provider "azurerm" {
  # if you're using a Service Principal (shared account) then either set the environment variables, or fill these in: 
  # subscription_id = "${var.subscription_id}"
  # client_id       = "${var.client_id}"
  # client_secret   = "${var.client_secret}"
  # tenant_id       = "${var.tenant_id}"
}

```

Reply:
ok, I will edit the pull request